privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -926,17 +926,18 @@ def testconnection(cls, param):
                     log.debug("{0!s}".format(traceback.format_exc()))
 
             if uidtype_count < count:  # pragma: no cover
-                desc = _("Your LDAP config found %i user objects, but only %i "
-                         "with the specified uidtype" % (count, uidtype_count))
+                desc = _("Your LDAP config found {0!s} user objects, but only {1!s} "
+                         "with the specified uidtype".format(count, uidtype_count))
```

Comment:
Oh wait -- shouldn't we call ``.format`` on the result of ``_("...")`` here? Right now, we first call ``.format()`` and pass this to ``_()``.